content/References/IPv6.md

```diff
@@ -0,0 +1,248 @@
+---
+title: "Understanding IPv6"
+description: "Provides information general information on IPv6 and implementation guidance."
+weight: 20
+tags:
+- network
+- interfaces
+- ipv6
+---
+
+## What is IPv6?
+IPv6 is the next generation Internet protocol replacing the current IPv4 protocol.
+IPv6 provides Pv6 provides 4294967296 times more addresses than IPv4.
```

Comment:
I screwed up.  It's not 4294967296.
It's actually 79228162514264337593543950336 times more addresses than IPv4.

Reply:
made a change here. Moved this line down and made it very generalized